docs/tutorials/understanding_21cmsense.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        hera.baselines_metres[:,:, 1].flatten());
</pre><p>the spacing here compared to the previous line looks odd.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/21cmSense/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>So it does. Fixed.</p>